scripts/xtensa-build-all.sh

```diff
@@ -233,31 +204,50 @@ do
 				XCC="none"
 				XTOBJCOPY="none"
 				XTOBJDUMP="none"
+				echo "XTENSA_TOOLS_DIR is not a directory"
 		fi
 	fi
 
 	# update ROOT directory for xt-xcc
 	if [ "$XCC" == "xt-xcc" ]
 	then
+		TOOLCHAIN=xt
 		ROOT="$XTENSA_BUILDS_DIR/$XTENSA_CORE/xtensa-elf"
 		export XTENSA_SYSTEM=$XTENSA_BUILDS_DIR/$XTENSA_CORE/config
 		PATH=$XTENSA_TOOLS_DIR/XtensaTools/bin:$OLDPATH
 	else
+		TOOLCHAIN=$HOST
 		PATH=$pwd/../$HOST/bin:$OLDPATH
 	fi
 
 	# only delete binary related to this build
 	rm -fr src/arch/xtensa/sof-$j.*
```

Comment:
This line is not needed.

Reply:
More like HOST is not needed, i need toolchain name that is 'xt' or 'xtensa-*-elf'

Reply:
Yes you are right.